lib/scenic/component/button.ex

```diff
@@ -278,11 +278,11 @@ defmodule Scenic.Component.Button do
     )
   end
 
-  defp do_special_theme_outline(graph, :dark, border) do
+  defp do_special_theme_outline(graph, {:scenic, :dark}, border) do
     Graph.modify(graph, :btn, &update_opts(&1, stroke: {1, border}))
   end
 
```

Comment:
I'm not sure about these, It doesn't seem like they will every match. I left them as is since I was already making such a sweeping change.